tests/sql/test_utf8_exprs.py

```diff
@@ -62,7 +62,8 @@ def test_utf8_exprs():
         normalize(a, remove_punct:=true, lowercase:=true) as normalize_remove_punct_lower_a,
         normalize(a, remove_punct:=true, lowercase:=true, white_space:=true) as normalize_remove_punct_lower_ws_a,
         tokenize_encode(a, 'r50k_base') as tokenize_encode_a,
-        tokenize_decode(tokenize_encode(a, 'r50k_base'), 'r50k_base') as tokenize_decode_a
+        tokenize_decode(tokenize_encode(a, 'r50k_base'), 'r50k_base') as tokenize_decode_a,
+        concat(a, '---') as concat_a,
```

Comment:
One more case you could add is concat with more than two arguments, since that should technically be supported too.